Cesium.Runtime/StdIoFunctions.cs

```diff
@@ -19,7 +19,9 @@ public static void PutS(byte* str)
         }
         catch (Exception) // TODO[#154]: Exception handling.
         {
+#pragma warning disable CS0219
```

Comment:
Personally, I would like to have this warning, so it will annoy me. That's guaranteed way to fix it in medium term

Reply:
Let's comment-out the `const int EOF` line instead.